libdnf/solv/pool.hpp

```diff
@@ -192,6 +192,8 @@ class Pool {
 
     bool is_installed(Id id) const { return is_installed(id2solvable(id)); }
 
+    bool is_excluded(Id id) const { return is_considered_map_active() && !considered.contains(id); }
```

Comment:
I know that it keeps uniformity but what about to add solvable into the name - like `is_solvable_excluded()`. Id is int and only valid range for the test is something like 1 to nsolvables. Please can you add a short description what is expected from Id value?

Reply:
OK renamed. Description added.